docs/content/concepts/logging.mdx

```diff
@@ -0,0 +1,91 @@
+---
+
+# Logging
+
+Dagster supports a variety of built-in logging options, as well as the ability to extend and customize them. By default, Dagster automatically tracks and captures all execution events, such as those during an asset materialization. During these events, logs like the following are generated:
+
+```bash
+12:40:05 - DEBUG - RUN_START - Started execution of run for "__ASSET_JOB_0".
+12:40:05 - DEBUG - ENGINE_EVENT - Executing steps using multiprocess executor: parent process (pid: 86387)
+12:40:05 - DEBUG - taxi_zones_file - STEP_WORKER_STARTING - Launching subprocess for "taxi_zones_file".
+12:40:07 - DEBUG - STEP_WORKER_STARTED - Executing step "taxi_zones_file" in subprocess.
+12:40:07 - DEBUG - taxi_zones_file - RESOURCE_INIT_S TARTED - Starting initialization of resources [io_manager].
+12:40:07 - DEBUG - taxi_zones_file - RESOURCE_INIT_SUCCESS - Finished initialization of resources [io_manager].
+12:40:07 - DEBUG - LOGS_CAPTURED - Started capturing logs in process (pid: 86390).
+12:40:07 - DEBUG - taxi_zones_file - STEP_START - Started execution of step "taxi_zones_file".
+12:40:09 - DEBUG - taxi_zones_file - STEP_OUTPUT - Yielded output "result" of type "Any". (Type check passed).
+12:40:09 - DEBUG - __ASSET_JOB_0 - taxi_zones_file - Writing file at: /Users/erincochran/Desktop/dagster-examples/project-dagster-university/tmpfxsoltsc/storage/taxi_zones_file using PickledObjectFilesystemIOManager...
+12:40:09 - DEBUG - taxi_zones_file - ASSET_MATERIALIZATION - Materialized value taxi_zones_file.
+12:40:09 - DEBUG - taxi_zones_file - HANDLED_OUTPUT - Handled output "result" using IO manager "io_manager"
+12:40:09 - DEBUG - taxi_zones_file - STEP_SUCCESS - Finished execution of step "taxi_zones_file" in 1.17s.
+12:40:09 - DEBUG - ENGINE_EVENT - Multiprocess executor: parent process exiting after 4.38s (pid: 86387)
+12:40:09 - DEBUG - RUN_SUCCESS - Finished execution of run for "__ASSET_JOB_0".
+```
+
+These logs can be exported to a local file or viewed in the [Dagster UI](#logs-in-the-dagster-ui).
+
+---
+
+## Benefits
+
+- Increased visibility into data pipelines
+- Easier debugging
+- Simplified identification of performance issues
+
+---
+
+## Uses
+
+To utilize logging in Dagster, you can:
+
+- [Extend and customize Dagster's built-in loggers](#built-in-loggers)
+- [Use Python's logging module](#python-logging)
+- [View and download logs in the Dagster UI](#dagster-ui)
+
+### Built-in loggers
+
+Dagster's [built-in loggers](/concepts/logging/loggers):
+
+- Support all levels of Python logs, such as `INFO`, `DEBUG`, `ERROR`, etc.
+- Can be configured to capture only specified levels, such as `ERROR`
+- Can include manually-defined messages produced inside certain Dagster definitions like assets, ops, and sensors
+
+**Loggers can also be customized**. Refer to the [Customizing loggers guide](/concepts/logging/custom-loggers) for more information and examples.
+
+### Python logging
```

Comment:
Should this section be marked as experimental?

Reply:
I wondered about that too, and about the current status of this feature in general. From what I could find it's been experimental for quite some time - I'll mark it as such, but any idea what we're doing with it?